docs/additional-configuration.adoc

```diff
@@ -225,6 +225,7 @@ you must add the following in your `~/.bashrc`:
 ----
 [ -f $HOME/ssh-environment ] && source $HOME/ssh-environment
 ----
+*Note:*  This is an experimental feature and is controlled by the `DevWorkspaceOperatorConfig.EnableExperimentalFeatures` option.
```

Comment:
Good call on documenting this! Maybe we should clarify that using an SSH passphrase is an experimental feature? Something like:
```adoc
*Note:*  Specifying a passphrase for an SSH key is an experimental feature and is controlled by the `DevWorkspaceOperatorConfig.EnableExperimentalFeatures` option.
```